packages/chaire-lib-backend/src/models/db/zones.db.queries.ts

```diff
@@ -164,13 +164,88 @@ const getZonesContaining = async (
     }
 };
 
+const addZonesAndConvertedGeography = async (
```

Comment:
This function deserves more comments. Why `addZonesAndConvertedGeography`? How is that different from the `create` or `createMultiple` functions? What is `srIdArray` and what is the expected format of the `geographyArray`?